lib/hexpm_web/router.ex

```diff
@@ -136,7 +142,7 @@ defmodule HexpmWeb.Router do
   end
 
   scope "/dashboard", HexpmWeb.Dashboard do
-    pipe_through :browser
+    pipe_through [:browser, :session, :defaults]
```

Comment:
Why do we separate the login and session pipelines for dashboard and the rest of the website?

Reply:
See above